mlrun/api/utils/auth/verifier.py

```diff
@@ -205,6 +205,9 @@ def authenticate_request(
             if "x-data-session-override" in request.headers:
                 auth_info.data_session = request.headers["x-data-session-override"]
 
+            if auth_info.user_unix_id is None:
+                auth_info.user_unix_id = iguazio_client.get_user_unix_id(request)
```

Comment:
This means another request to Iguazio per every request, it can cause a large performance hit, we should have a way to disable it